### DISCUSSION TITLE
General rtl of the template

### DISCUSSION THREAD

**Discussion by ghost** [ant-design-blazor/ant-design-blazor#3413] [General]
Hello and don't be tired to the antblazer team, I hope you will always be successful
In the rtl part of the project, I use the command ConfigProvider direction="rtl" in app.razor
I had a problem With this command, all sections are correctly specified as rtl, but the general format is still ltr
What should be done to change the general state of the template as follows?
![eng](https://github.com/ant-design-blazor/ant-design-blazor/assets/127301635/355c1815-31f9-4250-a7d2-334df5c815de)
![Persian](https://github.com/ant-design-blazor/ant-design-blazor/assets/127301635/5fef886d-bc03-4b8f-ac4b-8dfc934f4f1f)

**Comment by ElderJames** [comment]
Thanks for contacting us @Hadi69ans , this requires that the layout components support RTL. Some styles that belong to templates still need to be handled separately.

![image](https://github.com/ant-design-blazor/ant-design-blazor/assets/7550366/189c8c50-454a-49e6-b1b0-72c7f5602f70)

**Reply by ghost** [reply]
According to your words, it is not fixed yet, or will this issue be resolved with changes in Layout?
Does that mean we have to wait for the experts to solve the problem?
